packages/runtime/runtime-definitions/src/test/types/validateRuntimeDefinitionsPrevious.generated.ts

```diff
@@ -640,6 +640,18 @@ declare function use_old_InterfaceDeclaration_IGarbageCollectionDetailsBase(
 use_old_InterfaceDeclaration_IGarbageCollectionDetailsBase(
     get_current_InterfaceDeclaration_IGarbageCollectionDetailsBase());
 
+/*
```

Comment:
@ChumpChief - Tyler, Vlad, and I have all had local changes appear here that we've had to revert to pass CI.  I suspect you're encountering the same.

Reply:
Yep, good catch!